Dockerfile

```diff
@@ -1,15 +1,15 @@
 ARG NEXUS_VERSION=3.15.1
 
 FROM maven:3-jdk-8-alpine AS build
-ARG NEXUS_VERSION=3.15.1
+ARG NEXUS_VERSION=3.18.1
```

Comment:
why 3.18? and not latest one? 
But in general, after we include conan we will update this version also

Reply:
I think it should be 3.18 as was lust update in master. This is for old versions support. We did not cover old Conan version on top of 3.19 - 3.21. In 3.22 that we are working this is redundant

Reply:
we will update this version later.